src/views/footerView.jsx

```diff
@@ -264,42 +261,48 @@ export const ConferenceFooter = ({ isChatOpen, toggleChat }) => {
         centerComponents={[
           <AudioVideoToggle key="audioVideoToggle" />,
           isAllowedToPublish.screen && !isMobileDevice() ? (
-            <Button
-              key="toggleScreenShare"
-              iconOnly
-              variant="no-fill"
-              iconSize="md"
-              shape="rectangle"
-              classes={{ root: "mx-2" }}
-              onClick={() => toggleScreenShare(!isScreenShared)}
+            <Tooltip
+              title={`${!isScreenShared ? "Start" : "Stop"} screen sharing`}
             >
-              <ShareScreenIcon />
-            </Button>
+              <IconButton
+                active={!isScreenShared}
+                key="toggleScreenShare"
+                className="mx-2"
+                onClick={() => toggleScreenShare(!isScreenShared)}
+              >
+                <ShareScreenIcon />
+              </IconButton>
+            </Tooltip>
           ) : null,
           isAllowedToPublish.video && pluginRef.current?.isSupported() ? (
-            <Button
-              iconOnly
-              variant="no-fill"
-              shape="rectangle"
-              active={isVBPresent}
-              onClick={handleVirtualBackground}
-              classes={{ root: "mx-2" }}
-              key="VB"
+            <Tooltip
+              title={`Turn ${!isVBPresent ? "on" : "off"} virtual background`}
             >
-              <VirtualBackgroundIcon />
-            </Button>
+              <IconButton
+                active={!isVBPresent}
+                onClick={handleVirtualBackground}
+                className="mx-2"
+                key="VB"
+              >
+                <VirtualBackgroundIcon />
+              </IconButton>
+            </Tooltip>
           ) : null,
           isAllowedToPublish.audio && audiopluginRef.current?.isSupported() ? (
-            <Button
-              iconOnly
-              variant="no-fill"
-              shape="rectangle"
-              active={isNoiseSuppression}
-              onClick={handleNoiseSuppression}
-              key="noiseSuppression"
+            <Tooltip
+              title={`Turn ${
+                !isNoiseSuppression ? "on" : "off"
+              } noise suppression`}
             >
-              <NoiseSupressionIcon />
-            </Button>
+              <IconButton
```

Comment:
we can include tooltip in IconButton component itself, when tooltip/title prop is passed. In most cases, we would want a tooltip for IconButton's.